subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -500,7 +500,7 @@ static int __wifi_args_to_params(const struct shell *sh, size_t argc, char *argv
 				params->psk = optarg;
 				params->psk_length = strlen(params->psk);
 			} else {
-				PR_WARNING("Password configured without security configuration\n");
+				PR_WARNING("Passphrase provided without security configuration\n");
```

Comment:
This commit can be squashed in to the first one where the change was introduced.

Reply:
Done